parent/pom.xml

```diff
@@ -152,7 +152,11 @@
 				<artifactId>tycho-packaging-plugin</artifactId>
 				<version>${tychoVersion}</version>
 				<configuration>
-					<format>'${BUILD_ALIAS}-v'yyyyMMdd-HHmm</format>
+					<format>'${BUILD_ALIAS}-v'yyyyMMdd-HHmm'-LOCAL'</format>
+					<strictBinIncludes>false</strictBinIncludes>
+					<timestampProvider>jgit</timestampProvider>
+					<jgit.dirtyWorkingTree>warning</jgit.dirtyWorkingTree>
+					<jgit.ignore></jgit.ignore>
```

Comment:
In case we want to have pom.xml be ignored in future. Empty variable = nothing ignored.